js/ui/button.js

```diff
@@ -130,6 +130,20 @@ var Button = Widget.inherit({
             */
             useSubmitBehavior: false,
 
+            /**
+             * @name dxButtonOptions.toggleable
+             * @type boolean
+             * @default false
+             */
+            toggleable: false,
```

Comment:
Maybe better create a new widget, `ToggleButton` that extends Button? We can drop `toggleable` option and do not add more complexity to the Button

Reply:
I see no reason to create a new widget because of a difference between widgets in the "selected" option of the dxButton.

Reply:
I see at least one reason - `selected` option specific for the ToggleButton will be a part of the Button config. It can confuse our users

Reply:
I'm voting for the option. IMO we do not need a new widget if nobody needs it. Yes, the `selected` option could be confusing, but we already have options applied only if other options are set in other widgets.